Specs/Core/PolygonGeometrySpec.js

```diff
@@ -378,6 +378,64 @@ defineSuite([
         expect(p.indices.length).toEqual(numTriangles * 3);
     });
 
+    it('computes positions extruded and not closeTop', function() {
+        var p = PolygonGeometry.createGeometry(PolygonGeometry.fromPositions({
+             vertexFormat : VertexFormat.POSITION_ONLY,
+             positions : Cartesian3.fromDegreesArray([
+                                                         -1.0, -1.0,
+                                                         1.0, -1.0,
+                                                         1.0, 1.0,
+                                                         -1.0, 1.0
+                                                     ]),
+             extrudedHeight: 30000,
+             closeTop: false
+         }));
+
+        var numVertices = 37; // 13 bottom + 8 top edge + 8 bottom edge + 4 top corner + 4 bottom corner
+        var numTriangles = 32; // 16 bottom fill + 2 triangles * 4 sides
+        expect(p.attributes.position.values.length).toEqual(numVertices * 3);
+        expect(p.indices.length).toEqual(numTriangles * 3);
+    });
+
+    it('computes positions extruded and not closeBottom', function() {
+        var p = PolygonGeometry.createGeometry(PolygonGeometry.fromPositions({
+                                                                                 vertexFormat : VertexFormat.POSITION_ONLY,
```

Comment:
We should probably revert the Cesium-only tweak to the WebStorm defaults that is causing this to happen on format.  So annoying.